deepctr/layers/utils.py

```diff
@@ -30,9 +30,14 @@ class Hash(tf.keras.layers.Layer):
     if mask_zero = True,0 or 0.0 will be set to 0,other value will be set in range[1,num_buckets)
     """
 
-    def __init__(self, num_buckets, mask_zero=False, **kwargs):
+    def __init__(self, num_buckets, mask_zero=False, vocabulary_path=None, default_value=0, **kwargs):
```

Comment:
add some examples and docs about this feature

Reply:
I add some docs and one example in the doc,  please review it.